### PR TITLE
[15.0][ADD] portal_account_personal_data_only: display only own invoices in portal

### PR DESCRIPTION
**Pull request by yajo**
This module is half of the fix for https://github.com/OCA/sale-workflow/issues/2236.

Extracts only the account part of `portal_sale_personal_data_only` as it was in previous versions.

@moduon MT-1451 

Please review @fcvalgar

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
You'll need to adapt the tests to get rid of the residual sale stuff

**Review by fcvalgar — APPROVED** [review]
functional review ok

**Comment by yajo** [comment]
Thanks, tests fixed. Could you review again please @chienandalu?

**Review by rafaelbn — APPROVED** [review]
Thanks! 👍

**Review by Shide — APPROVED** [review]
Functional review OK

File: portal_account_personal_data_only/i18n/portal_sale_personal_data_only.pot [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline portal_account_personal_data_only/i18n/portal_sale_personal_data_only.pot]: This file can go out, and I suppose the translations as well.

**Review by chienandalu — APPROVED** [review]
Functional and code review :+1:

**Pull request merged**